be/src/udf/udf_internal.h

```diff
@@ -149,6 +152,7 @@ class FunctionContextImpl {
 
     // Empty if there's no error
     std::string _error_msg;
+    std::atomic<std::string*> _optional_error_msg{nullptr};
```

Comment:
anybody release the memory ? why not just a unique_ptr?

Reply:
_optional_error_msg only store nullptr or &_error_msg, it need not release memory